src/main/model-apps/super-res/super-res-inference-service.ts

```diff
@@ -77,6 +80,64 @@ class SuperResInferenceService implements InferenceService {
       }, 1000);
     });
   }
+
+  public async fetchModelInfo(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    server: ModelServerInfo,
+  ): Promise<ModelInfo | null> {
+    // return fetch(
+    //   `http://${server.serverAddress}:${server.serverPort}${MODEL_APP_INFO_SLUG}`,
+    // )
+    //   .then((res) => {
+    //     if (res.status !== 200) {
+    //       log.error(`Failed to fetch model info: ${res.statusText}`);
+    //       return null;
+    //     }
+    //     return res.json() as Promise<ModelInfo>;
+    //   })
+    //   .catch((error) => {
+    //     log.error(`Failed to fetch model info: ${error}`);
+    //     return null;
+    // });
+    return {
+      uid: 'model1',
+      name: 'Image Super Resolution',
+      version: '1.0.0',
+      author: 'John Doe',
+      lastUpdated: new Date('2023-10-01T12:00:00Z'),
+      description: 'This model upscales images to a higher resolution.',
+      parameters: [
+        {
+          name: 'Scale',
+          type: 'Number',
+          description:
+            'The factor by which to upscale the image (between 1 and 4).',
+        },
+      ],
+      inputTypes: [
+        {
+          type: 'Image(s)',
+          description: 'The images to be upscaled.',
+        },
+      ],
+      outputTypes: [
+        {
+          type: 'Image(s)',
+          description: 'The upscaled images.',
+        },
+      ],
+      constraints: [
+        {
+          name: 'Minimum Scale',
+          description: 'The minimum scale factor is 1.',
+        },
+        {
+          name: 'Maximum Scale',
+          description: 'The maximum scale factor is 4.',
+        },
+      ],
+    } as ModelInfo;
```

Comment:
you shouldn't be needing `as ModelInfo` if this object satisfies the type. If you're just trying to enforce it, you can use `} satisfies ModelInfo`.